### PR TITLE
debug: Prepend log macros with UVG_ 

### PR DESCRIPTION
**Pull request by devenvexe**
Prepend log macros with UVG_ to avoid conflicts with codebase that define their own LOG_* macros

### CODE REVIEW

**Comment by devenvexe** [comment]
Any chance to get a review on getting this merged? Happy to help how I can.

File: include/uvgrtp/debug.hh [hunk 1]
```diff
@@ -69,24 +69,24 @@ static inline void uvgrtp_debug(const char *level, const char *function, const c
...
+#define UVG_LOG_INFO(...) ;
+#define UVG_LOG_DEBUG(...) ;
 #else
-#define LOG_ERROR(...) uvgrtp_debug(LOG_LEVEL_ERROR, __func__, __VA_ARGS__)
-#define LOG_WARN(...)  uvgrtp_debug(LOG_LEVEL_WARN,  __func__, __VA_ARGS__)
-#define LOG_INFO(...)  uvgrtp_debug(LOG_LEVEL_INFO,  __func__, __VA_ARGS__)
+#define UVG_LOG_ERROR(...) uvgrtp_debug(LOG_LEVEL_ERROR, __func__, __VA_ARGS__)
+#define UVG_LOG_WARN(...)  uvgrtp_debug(LOG_LEVEL_WARN,  __func__, __VA_ARGS__)
+#define UVG_LOG_INFO(...)  uvgrtp_debug(LOG_LEVEL_INFO,  __func__, __VA_ARGS__)
+#define UVG_LOG_DEBUG(...)  uvgrtp_debug(LOG_LEVEL_DEBUG,  __func__, __VA_ARGS__)
```
Comment (jrsnen) [inline include/uvgrtp/debug.hh:80]: Both UVG_LOG_DEBUG should be modified into the LOG_DEBUG on line 66 and 68. This way, debug prints are disabled in release mode.

**Comment by jrsnen** [comment]
Hi and thank you for this PR. I found a small error in debug.hh, but otherwise looks good. I will probably end up moving debug.hh away from include soon anyway (internal debug prints don't need to be accessible to users of uvgRTP), but this change makes sense anyway.

**Comment by jrsnen** [comment]
If you can fix the error, I will merge this. If possible, modify the existing commit, but I will also accept one new commit to fix this.

**Comment by devenvexe** [comment]
Thank you I will address

**Pull request merged**